redis.conf

```diff
@@ -1160,17 +1157,26 @@ acllog-max-len 128
 # allkeys-lru -> Evict any key using approximated LRU.
 # volatile-lfu -> Evict using approximated LFU, only keys with an expire set.
 # allkeys-lfu -> Evict any key using approximated LFU.
+# volatile-lrp -> Evict using approximated LRP, only keys with an expire set.
+# allkeys-lrp -> Evict any key using approximated LRP.
 # volatile-random -> Remove a random key having an expire set.
 # allkeys-random -> Remove a random key, any key.
 # volatile-ttl -> Remove the key with the nearest expire time (minor TTL)
 # noeviction -> Don't evict anything, just return an error on write operations.
 #
 # LRU means Least Recently Used
 # LFU means Least Frequently Used
+# LRP means Least Recently Updated (only write operations update the timestamp)
```

Comment:
BTW, what does the `P` stands for? maybe it should be Least Recently Changed?

Reply:
or Modified

Reply:
augment choose the LRM